test/spark/app/.gitignore

```diff
@@ -0,0 +1,34 @@
+*.class
+*.log
+*.zip
+*.gz
+*.jpeg
+*.jpg
+*.png
```

Comment:
Not sure why we want to `.gitignore` these files.  They're all legitimate files (e.g. pictures in `resources/`).

Reply:
I'll use https://github.com/apache/spark/blob/master/.gitignore

Reply:
as you probably assumed I've used another spark project gitignore - but I'll update to use https://raw.githubusercontent.com/apache/spark/master/.gitignore